python/lsst/meas/extensions/simpleShape/__init__.py

```diff
@@ -1,5 +1,7 @@
+from lsst.meas.base import BasePlugin, wrapSimpleAlgorithm
```

Comment:
I suggest importing `lsst.meas.base` and then using that as a prefix for the symbols because this is an __init__.py file and any symbols you import here are imported into namespace `lsst.meas.extensions.simpleShape`. Another solution is to delete the symbols when you are done with them, but that's more error-prone.